integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml

```diff
@@ -48,7 +48,21 @@ spec:
       description: "A flag to enable or disable the Sealights integration feature. When set to 'true', test results are sent to Sealights for analysis; otherwise, this feature is skipped."
       default: "true"
   tasks:
+    - name: check-e2e-irrelevant-pr
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/jinqi7/konflux-qe-definitions
+          - name: revision
+            value: add_check_e2e_irrelevant_pr
+          - name: pathInRepo
+            value: tasks/check-e2e-irrelevant-pr/0.1/check-e2e-irrelevant-pr.yaml
```

Comment:
Can this be moved to some better location? Not only is it in a personal fork of https://github.com/konflux-ci/tekton-integration-catalog, but it's also a custom branch. How is this done in the catalog repo? Could we use a similar approach, or would that not apply here?

Reply:
It will be merged to https://github.com/konflux-ci/tekton-integration-catalog for sure. I'll change it to draft after the test is done also.

Reply:
Cool.

Reply:
Changed